index.js

```diff
@@ -75,6 +75,9 @@ function OptimizelyManager({ sdkKey, debug, ...rest }) {
 
   return {
     isFeatureEnabled(...args) {
+      // Check to see if we need to generate a userId
+      if(args[1] === undefined) args[1] = Math.random().toString()
```

Comment:
We could also use fancy ES6 default parameters:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters

`isFeatureEnabled(featureKey, userId = Math.random().toString())`

But I think your version is actually more clear